test/test_helper.rb

```diff
@@ -1,8 +1,8 @@
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
-require "codeclimate-test-reporter"
-CodeClimate::TestReporter.start
+require 'simplecov'
+SimpleCov.start
```

Comment:
https://docs.codeclimate.com/docs/travis-ci-ruby-test-coverage advised to put this at 'the very top' of the file.